src/wasm/wasm.cpp

```diff
@@ -56,6 +56,7 @@ const char* StringsFeature = "strings";
 const char* MultiMemoryFeature = "multimemory";
 const char* TypedContinuationsFeature = "typed-continuations";
 const char* SharedEverythingFeature = "shared-everything";
+const char* FP16Feature = "fp16";
```

Comment:
Just to double check, does this match the feature name used by LLVM in the custom features section?

Reply:
It will after https://github.com/llvm/llvm-project/pull/105434